### DISCUSSION TITLE
Hiring? Freelancers?

### DISCUSSION THREAD

**Discussion by emilsedgh** [juliangarnier/anime#996] [General]
I created this issue so it can be a hub for creating people who are willing to hire people with animejs experience or animejs engineers who like to be hired. A hub for contact exchange.

For start, if anyone is looking for freelance work, please let me know.

**Comment by chetanverma16** [comment]
@emilsedgh interested!!

**Comment by khajaamin** [comment]
@emilsedgh  - I am interested, Fullstack with React js and fabric js experience,  Now using anime with fabric js

**Comment by JediMindBang** [comment]
I actually run a legitimate coding school out of Manhattan, MERN with
Python (certificate and associates level). I would love to know more about
what you are looking for. Some of these graduates would jump at the chance!

Best,
Anthony

On Tue, Jun 21, 2022 at 4:15 PM Emil Sedgh ***@***.***> wrote:

> I created this issue so it can be a hub for creating people who are
> willing to hire people with animejs experience or animejs engineers who
> like to be hired. A hub for contact exchange.
>
> For start, if anyone is looking for freelance work, please let me know.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/juliangarnier/anime/issues/820>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGE2ZIWEC3TPKYNCXTVA2TLVQIPH3ANCNFSM5ZNUTXKA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Confidentiality Note: *This e-mail, and any attachment to it, contains
privileged and confidential information intended only for the use of the
individual(s) or entity named on the e-mail.  If the reader of this e-mail
is not the intended recipient, or the employee or agent responsible for
delivering it to the intended recipient, you are hereby notified that
reading it is strictly prohibited.  If you have received this e-mail in
error, please immediately return it to the sender and delete it from your
system.  Thank you.

**Comment by emilsedgh** [comment]
@khajaamin @JediMindBang Feel free to reach out to me on emil@rechat.com

**Comment by saminton** [comment]
Don't wanna be a downer but I feel this is kinda out of place, this is not a forum guys. There are many other places on the web better suited for this. These are meant to be issues concerning the repo. We can't start posting random stuff such as hiring offers. Think of the poor people trying to maintain this repo if they have to start going through tons of posts that doesn't concern them in the slightest.

**Comment by Pulse-Git** [comment]
> 

Maybe an Discord Server or something is coming in the future,
job offers can than moved to an specific channel.
I thought @juliangarnier announced something like that, but im not sure...

**Comment by sumitshinde-84** [comment]
Hello,
My name is Sumit Shinde, and I am a full-stack developer. I have recently completed my education and I'm actively seeking job opportunities to kickstart my career. While I don't have any professional experience yet, I have gained valuable skills and knowledge through my education and by contributing to open-source projects.

**Comment by raisaguys-cpu** [comment]
I got a win last night and it was real, I played on the JO777 site
